src/main/java/org/mule/extension/mulechain/internal/llm/type/LangchainLLMType.java

```diff
@@ -87,4 +94,21 @@ public String toString() {
       return this.value;
     }
   }
+
+  enum HuggingFaceModelName {
```

Comment:
[query] Don't we have any library provided enum class?

Reply:
It looks like huggingface supports 1 mil. models. Hence, it doesn't make sense to have them as a list.

Reply:
exactly